pybaseball/teamid_lookup.py

```diff
@@ -33,6 +33,26 @@ def team_ids(season: Optional[int] = None, league: str = 'ALL') -> pd.DataFrame:
     return fg_team_data
 
 
+def mlb_team_id(teamName=None):
+    '''
+        Provides team ids use within mlb.com urls
+
+        if no team is given all team ids are returned in a dataframe; else a single team id is returned as a string
+    '''
+    MLB_URL_FILE_NAME = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data', 'mlb_url_team_ID.csv')
+
+    mlb_url_team_ID = pd.read_csv(MLB_URL_FILE_NAME, index_col=0)
+    
+    if teamName is not None and len(teamName.strip()) != 0:
+        # Sanitize the input - remove spaces, dashes and make lower case
+        teamName = teamName.replace(" ", "").replace("-", "").lower()
+
+        mlb_url_team_ID = mlb_url_team_ID.query(f"team_name == '{teamName}'")
+        mlb_url_team_ID = mlb_url_team_ID['mlb_team_id'].loc[mlb_url_team_ID.index[0]]
+
+    return mlb_url_team_ID
```

Comment:
This is kind of nit-picky but I'm not sure I _love_ that this method returns either a string or a dataframe. The methods and approaches you would do to a string are pretty different from that you would do on a DF. IMO I'd add a second function that just returns just the DF and then let `mlb_team_id` slice the DF to the passed teamName

Reply:
@tjburch I initially just used the PR/code changes that someone else had submitted so we could get the fix out. I agree that function needed some work. So, I went ahead and cleaned it up. It will now either return a number such as 112 (if the string Cubs is passed it) or throw an error if the team name isn't found.